tests/conftest.py

```diff
@@ -4622,8 +4622,12 @@ def finalizer():
         """
         if len(KMS.get_encryption_kmsid()) > 1:
             KMS.remove_kmsid(vault.kmsid)
-        vault.remove_vault_backend_path()
-        vault.remove_vault_policy()
+        if vault.vault_namespace:
+            vault.remove_vault_backend_path(vault_namespace=vault.vault_namespace)
+            vault.remove_vault_policy(vault_namespace=vault.vault_namespace)
+        else:
+            vault.remove_vault_backend_path(vault_namespace=None)
+            vault.remove_vault_policy(vault_namespace=None)
```

Comment:
You don't have to pass the vault_namespace parameter here

Reply:
Yes, right